.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,7 +2,15 @@
 Thank you for your Pull Request. Please provide a description above and review
 the requirements below.
 
-Bug fixes and new features should include tests and possibly benchmarks.
-
 Contributors guide: https://github.com/electron/electron/blob/master/CONTRIBUTING.md
--->
\ No newline at end of file
+-->
+
+##### Checklist
+<!-- Remove items that do not apply. For completed items, change [ ] to [x]. -->
+
+- [ ] PR description included and stakeholders cc'd
+- [ ] `npm test` passes
+- [ ] `npm run lint` passes
```

Comment:
If `test` is passing, then `lint` is also necessarily passing. Maybe these should be reversed?

Reply:
i can remove lint then i think